guide.html

```diff
@@ -0,0 +1,425 @@
+                <p class="lead" data-i18n="dgc-language">Language</p>
+                <a class="text-white select-lang" style="cursor: pointer;" data-lang="en">English</a><br>
+                <a class="text-white select-lang" style="cursor: pointer;" data-lang="de">Deutsch</a><br>
+                <a class="text-white select-lang" style="cursor: pointer;" data-lang="fr">Français</a><br>
+                <a class="text-white select-lang" style="cursor: pointer;" data-lang="es">Español</a><br>
+            </div>
+        </div>
+    </div>
+</footer>
+<!-- Copyright Section-->
+<div class="copyright py-4 text-center text-white">
+    <div class="container"><small>Copyright © The Ðogecoin Project 2021</small></div>
+</div>
+<!-- Scroll to Top Button (Only visible on small and extra-small screen sizes)-->
+<div class="scroll-to-top d-lg-none position-fixed">
+    <a class="js-scroll-trigger d-block text-center text-white rounded" href="#page-top"><i class="fa fa-chevron-up"></i></a>
+</div>
+<!-- Bootstrap core JS-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js"></script>
+<!-- Third party plugin JS-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js"></script>
+<script src="assets/vendor/js/particles.min.js"></script>
+    <!-- Internationalization -->
+    <script src="assets/vendor/js/jquery.i18n/jquery.i18n.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/jquery.i18n.messagestore.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/jquery.i18n.fallbacks.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/jquery.i18n.language.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/jquery.i18n.parser.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/jquery.i18n.emitter.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/bs.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/dsb.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/fi.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/ga.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/he.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/hsb.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/hu.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/hy.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/la.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/ml.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/os.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/ru.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/sl.js"></script>
+    <script src="assets/vendor/js/jquery.i18n/languages/uk.js"></script>
```

Comment:
do all of these need to be loaded even if we don't have translations? If yes, can we minify this into a single script?

Reply:
no these files where never used (imho only needed for special characters), just included them for the sake of completeness (as they are included in the current live page). removed them now